cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/paymentOptions/alternativePaymentMethodSummary.isml

```diff
@@ -1,3 +1,23 @@
 <!--- TEMPLATENAME: alternativePaymentMethodSummary.isml --->
 <iscontent compact="true"/>
+<isset name="AdyenHelper" value="${require('*/cartridge/scripts/util/AdyenHelper')}" scope="pdict"/>
+<script src="${pdict.AdyenHelper.getCheckoutUrl()}" type="text/javascript"></script>
+<link rel="stylesheet" type="text/css" href="${pdict.AdyenHelper.getCheckoutCSS()}"/>
+
+<isif condition="${payment.adyenAction != null}">
+    <!--- Payment action for Voucher --->
+    <div id="voucherResult"></div>
+    <br/>
+    <script type="text/javascript">
+        var configuration = {
+            locale: '${request.locale}'
+        };
+
+        const checkout = new AdyenCheckout(configuration);
+        const voucherResult = document.getElementById('voucherResult');
+        var action = JSON.parse('${payment.adyenAction}'.replace(/&quot;/g, '\"'));
```

Comment:
looks a bit nasty with .replace(/&quot;/g, '\"') is there no better solution possible ? @KadoBOT can you have a look as well ?

Reply:
No other solution